acrobat/scripts/utils.js

```diff
@@ -25,8 +25,9 @@ export const [setLibs, getLibs] = (() => {
     (prodLibs, location) => {
       libs = (() => {
         const { hostname, search } = location || window.location;
+        // eslint-disable-next-line compat/compat
         const branch = new URLSearchParams(search).get('milolibs') || 'main';
-        if (branch === 'main' && hostname === 'www.stage.adobe.com') return 'https://www.adobe.com/libs';
+        if (branch === 'main' && hostname === 'www.stage.adobe.com') return 'https://stage.adobe.com/libs';
         if (!(hostname.includes('.hlx.') || hostname.includes('local') || hostname.includes('stage'))) return prodLibs;
         if (branch === 'local') return 'http://localhost:6456/libs';
         return branch.includes('--') ? `https://${branch}.hlx.page/libs` : `https://${branch}--milo--adobecom.hlx.page/libs`;
```

Comment:
Please make this the same as what we have in scripts.js. 
https://github.com/adobecom/dc/blob/32b4f6c3b37e6bdece7c30a79d8b71f29f68f13c/acrobat/scripts/scripts.js#L31